src/utilities/createHabitData.ts

```diff
@@ -0,0 +1,36 @@
+import lightFormat from "date-fns/lightFormat";
+
+export function createHabitData() {
+  const habitData = [];
+  const rewardsDays = [
+    Math.ceil(Math.random() * 30),
+    Math.ceil(Math.random() * 30),
+    Math.ceil(Math.random() * 30),
+    Math.ceil(Math.random() * 30),
+    Math.ceil(Math.random() * 30),
+    Math.ceil(Math.random() * 30),
+  ];
+  const startDate = new Date(Date.now());
+
+  for (let i = 1; i <= 30; i++) {
+    const currentDate = new Date();
+    currentDate.setDate(startDate.getDate() + i - 1);
+
+    if (rewardsDays.includes(i)) {
+      habitData.push({
+        id: i,
```

Comment:
why the `id` is needed for the habit data as you already have the `date` property?